src/corelibs/U2Designer/src/DelegateEditors.h

```diff
@@ -352,6 +357,7 @@ private slots:
 
 protected:
     QVariantMap items;
+    QVariantMap visibleKeyToBusValueTranslationMap;
```

Comment:
Why do you have QVariant as a value here? Looks like it can be a QString->QString map?